src/pages/UnlockedWallet/OverviewPage/AssetsOverviewPanel.tsx

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { convertSetToFiat } from '@alephium/sdk'
+import classNames from 'classnames'
+import { ArrowDown, ArrowUp, Lock, Settings } from 'lucide-react'
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import Amount from '@/components/Amount'
+import Button from '@/components/Button'
+import { useAddressesContext } from '@/contexts/addresses'
+import { useGlobalContext } from '@/contexts/global'
+import { useAppSelector } from '@/hooks/redux'
+import ModalPortal from '@/modals/ModalPortal'
+import SendModalTransfer from '@/modals/SendModals/SendModalTransfer'
+import SettingsModal from '@/modals/SettingsModal'
+import { useGetPriceQuery } from '@/store/priceApiSlice'
+import { currencies } from '@/utils/currencies'
+
+interface AssetsOverviewProps {
+  isLoading?: boolean
+  className?: string
+}
+
+const AssetsOverview = ({ className, isLoading }: AssetsOverviewProps) => {
```

Comment:
Nit: based on the name, I thought this would be the "assets / tokens" card. What about `WalletAmountsOverview`, `WalletOverviewPanel` or something similar?

Reply:
I renamed `AssetsOverviewPanel` to `AmountsOverviewPanel` :) Hope that's fine!